quartz/plugins/transformers/ofm.ts

```diff
@@ -171,27 +181,34 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
           src = src.toString()
         }
 
+        // replace all wikilinks inside a table first
+        src = src.replace(tableRegex, (value) => {
+          // escape all aliasses and headers in wikilinks inside a table
+          return value.replace(tableWikilinkRegex, (value, ...capture) => {
+            const [raw]: (string | undefined)[] = capture
+            let escaped = raw ?? ""
+            escaped = escaped.replace(/\#/g, "\\#")
```

Comment:
is there a reason to double escape the # ? iirc we only double escape the pipe because its a table character

Reply:
If I don't escape the `#`, then all header wikilinks (so `[[link#header]]` or `[[link#header|alias]]` don't parse at all inside tables.

The single escape is not actually an escape. It's to ensure the regex matches the `#` character properly.

That being said, I could probably just use a regular replace (non-regex) instead.